src/VisualizationTests/WpfVisualizationTests.csproj

```diff
@@ -188,6 +188,9 @@
   <ItemGroup>
     <WCFMetadata Include="Service References\" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
Reverting so they can be dealt with separately.
https://github.com/nunit/nunit-vs-adapter/issues/51